### PR TITLE
Clarify that rootMargin is only applied to root.

### PR DESCRIPTION
**Pull request by szager-chromium**
Issue #70

### CODE REVIEW

**Comment by mpb** [comment]
|options.rootMargin| doesn't look correct in the bikeshed output.

File: index.bs [hunk 1]
```diff
@@ -174,11 +174,9 @@ interface IntersectionObserver {
 		3. Set |this|.{{IntersectionObserver/root}} to
 			|options|.{{IntersectionObserverInit/root}}
 		4. Attempt to <a>parse a root margin</a>
-			from |options|.{{IntersectionObserverInit/rootMargin}}.
-			If a list is returned,
+			from |options.rootMargin|.  If a list is returned,
```
Comment (mpb) [inline index.bs]: This line doesn't seem to translate correctly into _options.rootMargin_ in the output html.

Comment (szager-chromium) [inline index.bs]: Fixed, also added some more verbiage to the threshold attribute description.

**Comment by szager-chromium** [comment]
ping

**Comment by ojanvafai** [comment]
lgtm

**Pull request merged**